snowfakery/api.py

```diff
@@ -114,6 +114,12 @@ def check_if_finished(self, id_manager):
 
         return last_used_id >= target_id
 
+    def configure_output_stream(
```

Comment:
Puts the parent application in charge of the output stream creation. By default it creates the same output streams it always did.